misc/dbt-materialize/dbt/include/materialize/macros/adapters.sql

```diff
@@ -151,6 +159,25 @@
   {% endif %}
 {% endmacro %}
 
+{% macro materialize__get_refresh_interval_sql(relation, refresh_interval_dict) -%}
+  {%- set refresh_interval = adapter.parse_refresh_interval(refresh_interval_dict) -%}
+    {% if refresh_interval.at -%}
+      refresh at '{{ refresh_interval.at }}'
+    {%- endif %}
+    {% if refresh_interval.at_creation -%}
+      refresh at creation
+    {%- endif %}
+    {% if refresh_interval.every -%}
+      {% if refresh_interval.at or refresh_interval.at_creation -%}
+        ,
+      {%- endif %}
+      refresh every '{{ refresh_interval.every }}'
+      {% if refresh_interval.aligned_to -%}
+        aligned to '{{ refresh_interval.aligned_to }}'
+      {%- endif %}
+    {%- endif %}
+{%- endmacro %}
```

Comment:
If I understand correctly, this only supports specifying each of the options a maximum of 1 time. Materialize actually allows specifying `REFRESH AT` and `REFRESH EVERY` multiple times. But I think this is not urgently needed, so it's ok to fix this in a follow-up PR.

> at and at_creation are exclusive

They are not exclusive, so it's ok that this dbt code allows for specifying them together.

Reply:
Ah! I didn't realize these scenarios were possible. That will slightly change the syntax, though, so there's possibly some annoyance in shipping it as a quick follow-up. I'll keep it in the backburner!